cnd/src/http_api/markets/get_btc_dai.rs

```diff
@@ -1,12 +1,13 @@
 use crate::{
-    http_api::{problem, serde_peer_id, Amount},
+    http_api::{problem, Amount},
     network::Swarm,
+    serde,
 };
+use ::serde::Serialize;
```

Comment:
Should we re-export everything from the `serde` crate from the `serde` module so we can avoid this?